py-polars/polars/convert.py

```diff
@@ -423,8 +423,30 @@ def from_numpy(
     )
 
 
+@overload
+def from_arrow(
+    data: pa.Array | pa.ChunkedArray,
+    schema: SchemaDefinition | None = ...,
+    *,
+    schema_overrides: SchemaDict | None = ...,
+    rechunk: bool = ...,
+) -> Series:
+    ...
+
+
+@overload
+def from_arrow(  # type: ignore[misc]
```

Comment:
Unfortunately, overload doesn't work here. pyarrow does not support type hints, so the first overload entry is `Any` and it will match everything. So I think this must be reverted.

Reply:
"Ouch" ... you're quite right - I even noted that on an earlier issue myself! Will revert that part and add a note about it next to the function so we don't do it a third time 🤣

Reply:
Yea I should have left a comment 😬 sorry about that. Would be great if you could add a note indeed!

Reply:
Done... patch incoming 🍾🤣